__integrations__/index.test.ts

```diff
@@ -0,0 +1,57 @@
+import { aFiscalCode, aMessageContent } from "../__mocks__/mocks";
+import { WAIT_MS } from "./env";
+import {
+  Client as FnServicesClient,
+  createClient
+} from "./generated/fn-services/client";
+
+import * as E from "fp-ts/Either";
+
+jest.setTimeout(WAIT_MS * 5);
+
+const fetch = require("node-fetch");
+const baseUrl = "http://function:7071";
+
+beforeAll(async () => {});
+afterAll(async () => {});
+
+beforeEach(() => jest.clearAllMocks());
+
+const aSubscriptionKey = "aSubscriptionKey";
+const aContent = aMessageContent;
+
+describe("fn-services |> ping", () => {
```

Comment:
I'm not sure about the added value of this kind of integration tests. 
They require a lot of setup and moving parts. What are we testing here?
Under which conditions a unit tested endpoint could return a result different to what's expected?

Reply:
I'm just setting up the environment, then I'll add message creation tests. 
We are going to refactor that process, so we need to test it

Reply:
> [...]They require a lot of setup and moving parts [...] Under which conditions a unit tested endpoint could return a result different to what's expected?

@gunzip the value is to ensure all moving parts are cooperating properly. We don't expect such kind of integration tests to cover all the spectrum of possible cases, we just want to ensure a workflow which involves more than a function can be processed as a whole.